common/sw-update/versions-db-manager.h

```diff
@@ -15,78 +15,91 @@ namespace rs2
 
     namespace sw_update
     {
-        // The version_db_manager class download, parse and supply queries for the RS components versions information.
-        // The version file can be stored locally on the file system or on a HTTP server.
-        class versions_db_manager
+        enum update_policy_type { EXPERIMENTAL, RECOMMENDED, ESSENTIAL };
+        enum component_part_type { LIBREALSENSE, VIEWER, DEPTH_QUALITY_TOOL, FIRMWARE };
+        enum update_source_type { FROM_FILE, FROM_SERVER };
+        enum query_status_type { VERSION_FOUND, NO_VERSION_FOUND, DB_LOAD_FAILURE };
+
+
+        std::string to_string(const component_part_type& component) ;
+        std::string to_string(const update_policy_type& policy) ;
+        bool from_string(std::string component_str, component_part_type& component_val) ;
```

Comment:
const &, on this and next line

Reply:
And please remove the spaces before the `;`

Reply:
👍